editor/src/messages/portfolio/document/node_graph/node_graph_message_handler/node_properties.rs

```diff
@@ -377,23 +363,21 @@ pub fn imaginate_properties(document_node: &DocumentNode, node_id: NodeId, conte
 		let widgets = vec![
 			WidgetHolder::text_widget("Server"),
 			WidgetHolder::unrelated_separator(),
-			WidgetHolder::new(Widget::IconButton(IconButton {
-				size: 24,
-				icon: "Settings".into(),
-				tooltip: "Preferences: Imaginate".into(),
-				on_update: WidgetCallback::new(|_| DialogMessage::RequestPreferencesDialog.into()),
-				..Default::default()
-			})),
+			IconButton::new()
```

Comment:
And IconButton should probably take its icon name and size since those are mandatory. Likewise for IconLabel.

Reply:
Fair enough, I'll change this.